pkg/datasources/system_get_aws_sns_iam_policy_acceptance_test.go

```diff
@@ -17,17 +18,17 @@ func providers() map[string]*schema.Provider {
 }
 
 func TestAccSystemGetAWSSNSIAMPolicy_basic(t *testing.T) {
-	// r := require.New(t)
-
+	// IAM user in the Principal part is unique for each Snowflake account.
+	// This just check it policy json has a valid form.
+	r, _ := regexp.Compile(`\{"Version":"2012\-10\-17","Statement":\[\{"Sid":"1","Effect":"Allow","Principal":\{"AWS":"arn:aws:iam::[0-9]+:user\/[0-9a-zA-Z_\+=,\.@\-]+"\},"Action":\["sns:Subscribe"\],"Resource":\["arn:aws:sns:us\-east\-1:1234567890123456:mytopic"\]\}\]\}`)
```

Comment:
I think parsing json with a regex like this is too brittle.

Reply:
what do you want? you can suggest

Reply:
I don't think we can depend much of the content, other than it is a string that contains JSON.

So I think something as simple as `a non-empty` string would be fine.